tools/doc/package.json

```
@@ -7,18 +7,18 @@
     "node": ">=12.10.0"
```

We're using TLA in addon-verify
```suggestion
    "node": ">=14.8.0"
```